app/actions/GroupActions.ts

```diff
@@ -43,6 +44,25 @@ export function removeMember(membership: Record<string, any>): Thunk<any> {
     },
   });
 }
+export function editMembership(
+  membership: MembershipType,
+  role: string
```

Comment:
Do we have an enum for this?

Reply:
Ah yes you're right!

Reply:
well, it's actually a `const`. Would you like something like `role: keyof typeof ROLES`? I see that `role: string` is the way that it's done elsewhere, so I would need to rewrite it like that multiple places.

Reply:
I typed it correctly now 👍🏼